server/api.go

```diff
@@ -634,3 +635,27 @@ func mapChildNamespaceRoute(route martini.Router, namespace *schema.Namespace) {
 		},
 	)
 }
+
+func mapHealthcheckRoute(route martini.Router, manager *schema.Manager, healthcheck *healthcheck.Healthcheck) {
+	healthcheckSchema, hasSchema := manager.Schema("healthcheck")
+	if !hasSchema {
+		panic("The 'healthcheck' schema is missing. Check if gohan.json is loaded")
+	}
+	url := healthcheckSchema.GetPluralURL()
+
+	log.Debug("Registering the healthcheck handler on %s", url)
+
+	get := func(w http.ResponseWriter, params martini.Params, auth schema.Authorization) {
+		defer resources.MeasureRequestTime(time.Now(), "get", healthcheckSchema.ID)
+
+		if err := healthcheck.IsHealthy(); err == nil {
+			http.Error(w, "", http.StatusOK)
+		} else {
+			log.Error("healthcheck error:" + err.Error())
```

Comment:
please use `Errorf` for log messages with arguments (if the underlying implementation is correct, it will be more efficient)

Reply:
changed format